components/hab/src/command/pkg.rs

```diff
@@ -184,17 +184,24 @@ pub mod export {
 
         pub fn format_for(_ui: &mut UI, value: &str) -> Result<ExportFormat> {
             match value {
-                "docker" => {
+                "aci" => {
                     let format = ExportFormat {
-                        pkg_ident: try!(PackageIdent::from_str("core/hab-pkg-dockerize")),
-                        cmd: "hab-pkg-dockerize".to_string(),
+                        pkg_ident: try!(PackageIdent::from_str("core/hab-pkg-aci")),
+                        cmd: "hab-pkg-aci".to_string(),
                     };
                     Ok(format)
                 }
-                "aci" => {
+                "deb" => {
```

Comment:
I'm not very familiar with `deb` packages, other than the limited understanding I've gained recently while researching package managers. That said, I agree that we should have a discussion about _when/why_ to build upon the package export feature since, as you stated above, we anticipate the need for even more image formats.